src/client/Client.cc

```diff
@@ -3535,6 +3535,7 @@ int Client::get_caps_used(Inode *in)
 void Client::cap_delay_requeue(Inode *in)
 {
   ldout(cct, 10) << __func__ << " on " << *in << dendl;
+
   in->hold_caps_until = ceph::coarse_mono_clock::now() + cct->_conf.get_val<std::chrono::seconds>("client_caps_release_delay");
```

Comment:
You have defined a `caps_release_delay`, you should use it here to improve the perf.

Reply:
Replaced